08-projects.Rmd

```diff
@@ -5126,21 +5159,23 @@ id | name | capacity | address
 2  | Cary Quadrangle | 1200 | 1016 W Stadium Ave, West Lafayette, IN 47906
 3  | Hillenbrand Hall | NULL | 1301 3rd Street, West Lafayette, IN 47906
 
-**Relevant topics:** [sql](#sql), [inner join](#sql-inner-join)
+**Hint:** You can find how to typeset tables in the [RMarkdown cheatsheet](https://rstudio.com/wp-content/uploads/2016/03/rmarkdown-cheatsheet-2.0.pdf).
+
+**Relevant topics:** [SQL](#sql), [INNER JOIN](#sql-inner-join)
 
 ```{block, type="bbox"}
 **Item(s) to submit:**
 
 - [RMarkdown](#r-rmarkdown) table displaying the result of performing the following query as a table.
 ```
 
-##### 2. Using the same two tables from (1), use [RMarkdown](#r-rmarkdown) to display the result of performing the following query as a table. Explain the difference between an INNER JOIN and LEFT JOIN.
+##### 2. Using the same two tables from (1), use [RMarkdown](#r-rmarkdown) to manually type and display the result of performing the following query as a table. Explain the difference between an INNER JOIN and LEFT JOIN.
 
 ```{sql, eval=F}
 SELECT * FROM users AS u LEFT JOIN dorms AS d ON u.dorm=d.id;
 ```
 
-**Relevant topics:** [sql](#sql), left join
+**Relevant topics:** [SQL](#sql), LEFT JOIN
```

Comment:
@kevinamstutz , similar to the INNER JOIN link, could you double check the reference section for LEFT JOIN? Thanks!

Reply:
Same deal as before, I'll add/fix anchor links in this project soon.